apps/api/src/routes/application-router.ts

```diff
@@ -295,4 +296,66 @@ applicationRouter.post('/applications/reject', jsonParser, async (req, res) => {
 		});
 	}
 });
+
+applicationRouter.post('/applications/submit', jsonParser, async (req, res) => {
+	const { applicationId, role, signature } = req.body;
+
+	if (!applicationId || !role || !signature) {
+		res.status(400).json({ message: 'Missing required fields.' });
+	}
+
+	if (!applicationId || isNaN(parseInt(applicationId))) {
```

Comment:
`!applicationId` is already checked in the above conditional
Also we can replace `isNaN(parseInt(applicationId)` with `isPositiveNumber` from `/utils/routes`